ast-visual-studio-extension/CxExtension/Commands/AddExtensionWindowCmd.cs

```diff
@@ -42,16 +44,27 @@ public static async Task InitializeAsync(AsyncPackage package)
             Instance = new CxWindowCommand(package, commandService);
         }
 
-        private void Execute(object sender, EventArgs e)
+        public void Execute(object sender, EventArgs e)
         {
             _ = this.package.JoinableTaskFactory.RunAsync(async delegate
-              {
-                  ToolWindowPane window = await this.package.ShowToolWindowAsync(typeof(CxWindow), 0, true, this.package.DisposalToken);
-                  if ((null == window) || (null == window.Frame))
-                  {
-                      throw new NotSupportedException("Cannot create tool window");
-                  }
-              });
+            {
+                try
+                {
+                    await Task.Delay(1000);
+                    ToolWindowPane window = await this.package.ShowToolWindowAsync(typeof(CxWindow), 0, true, this.package.DisposalToken);
+                    if ((null == window) || (null == window.Frame))
+                    {
+                        throw new NotSupportedException("Cannot create tool window");
+                    }
+                    // Explicitly show the window
+                      var frame = (IVsWindowFrame)window.Frame;
+                      Microsoft.VisualStudio.ErrorHandler.ThrowOnFailure(frame.Show());
+                }
+                catch (Exception ex)
+                {
+                    Debug.WriteLine($"Failed to show window: {ex}");
```

Comment:
should we also add this log to the new log file?

Reply:
We'll first wait for the fix related to Sara addition regarding the logs, and then there will be a task to include everything in the logs.